src/js/jsx/sections/style/VectorFill.jsx

```diff
@@ -181,44 +181,52 @@ define(function (require, exports, module) {
                     document={this.props.document}
                     layers={this.state.layers}
                     fill={this.state.fill} />);
+
+            if (this.props.document.layers.selected) {
+                var disabledVectorFill = this.props.document.layers.selected.filter(function (layer) {
```

Comment:
This is legal and functionally correct (due to [variable hoisting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting)), but for clarity I think we would prefer something like this:
```javascript
var disabledVectorFill;
if (this.props.document.layers.selected.isEmpty()) {
    disabledVectorFill = false;
} else {
    disabledVectorFill = // filter...
}
```